Dangerfile.swift

```diff
@@ -174,7 +175,7 @@ extension DangerDSL {
     
     enum BranchType {
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Declaration maste...
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 177;
-->

- :warning: Declaration master contains the term "master" which is not considered inclusive. (`inclusive_language`)